go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/anacrolix/args v0.4.1-0.20211104085705-59f0fe94eb8f
 	github.com/anacrolix/chansync v0.3.0
 	github.com/anacrolix/confluence v1.9.0 // indirect
-	github.com/anacrolix/dht/v2 v2.13.0
+	github.com/anacrolix/dht/v2 v2.13.1-pr54-fix-marshal
```

Comment:
This needs to be updated when anacrolix/dht#54 is merged and received a version tag. This PR can't be merged until then.